# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../lm_sensors/lm_sensors-3.3.0-f71882fg-support.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Add Fintek F71889A and F81865F support
Origin: Upstream
http://www.lm-sensors.org/changeset/5952

Index: /lm-sensors/trunk/prog/detect/sensors-detect
===================================================================
--- trunk/prog/detect/sensors-detect	(revision 5946)
+++ trunk/prog/detect/sensors-detect	(revision 5952)
@@ -1994,5 +1994,5 @@
 	}, {
 		name => "Fintek F71889A Super IO Sensors",
-		driver => "to-be-written",
+		driver => "f71882fg",
 		devid => 0x1005,
 		logdev => 0x04,
@@ -2008,5 +2008,5 @@
 	}, {
 		name => "Fintek F81865F Super IO Sensors",
-		driver => "to-be-written",
+		driver => "f71882fg",
 		devid => 0x0704,
 		logdev => 0x04,
